cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -1580,15 +1591,15 @@ def test_colors_not_numpy_array(self, validator_with_adata):
 
     def test_colors_bool_obs_counterpart(self, validator_with_adata):
         validator = validator_with_adata
-        validator.adata.uns["is_primary_data_colors"] = ["green", "purple"]
+        validator.adata.uns["is_primary_data_colors"] = numpy.array(["green", "purple"])
```

Comment:
not related to the primary change, but i updated these two test cases to be numpy arrays since while they should fail for having invalid `obs` counterparts, the colors should be valid numpy arrays